ofl/caramel/METADATA.pb

```diff
@@ -0,0 +1,18 @@
+name: "Caramel"
+designer: "Robert Leuschke"
+license: "OFL"
+category: "DISPLAY"
```

Comment:
HANDWRITING would fit better no? Playball has this issue too (is the only one as far as I see)

Reply:
> HANDWRITING would fit better no? Playball has this issue too (is the only one as far as I see)

Oh yes, thanks! Done.